enterprise_knowledge_retriever/src/document_retrieval.py

```diff
@@ -149,7 +149,6 @@ def set_llm(self):
                     "max_tokens_to_generate": self.llm_info["max_tokens_to_generate"],
                     "temperature": self.llm_info["temperature"],
                     "process_prompt": True,
-                    "select_expert": self.llm_info["select_expert"]
```

Comment:
removed the wrong line,  here was the process prompt line, not the select expert one

Reply:
@snova-jorgep have updated! Thanks